c.html.markdown

```diff
@@ -326,7 +326,7 @@ int main (int argc, char** argv)
   using "goto" in C
   */
   typedef enum { false, true } bool;
-  // for C don't have bool as data type :(
+  // C doesn't have bool as a data type :(
   bool disaster = false;
```

Comment:
I think
> for C don't have bool as data type

and
> C doesn't have bool as a data type

are alike.


If we are talking about C99 or later instead of ANSI C, _Bool and <stdbool.h> should be mentioned.